0001-kaa-protocol/README.md

```diff
@@ -159,6 +162,10 @@ In that case, the endpoint is manually added to the server, the server allocates
 It's good to keep topic names short.
 Maybe we can introduce topic aliases, so user won't need to use full-length topic name.
 
+### Status topics
+Would it make sense to move `/status` suffix to some place in the middle of the MQTT path (like between the common and extension-specific parts) instead of the end?
```

Comment:
If it is, it should be placed right after the endpoint token. In that case, a wildcard to subscribe to all endpoint responses is `kaa/+/+/<endpoint_token>/status/#`.

Downsides are:
- The rule seems to be more complex. When it is "whatever the request topic is, just append /status", it is really easy to follow.
- Status topics have different directions. (Some are client->server, some are server->client.) That means that subscribing to all status topics isn't always a good idea.